projects/plugins/jetpack/extensions/blocks/seo/components/placeholder.scss

```diff
@@ -0,0 +1,3 @@
+.components-placeholder__learn-more {
```

Comment:
Should this be more specific to avoid targetting others containers with the same class in the editor? 

Or maybe we should go the opposite, and start using a common, Jetpack-unique class for all those placeholders, and then style it all from one place?

Reply:
Yes, good point. I think for now I will update the naming to be more specific. 
But after we finish up with all the blocks, I think we should create a generic component and reuse it.